ods_ci/tests/Resources/Files/machineset-template.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Have you considered creating Machine set from scratch as described in https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-aws.html ?